src/lib/utilities/format-date.ts

```diff
@@ -149,3 +150,23 @@ export function getMilliseconds(date: ValidTime | undefined | null): number {
 
   return getSecondAsMilliseconds(parsedDate);
 }
+
+export function fromSecondsToDaysOrHours(seconds: string): string {
```

Comment:
I think we can just used intervalToDuration.

Reply:
Updated to use intervalToDuration -> durationToString